notebooks/video_clips/VideoClips.ipynb

```diff
@@ -0,0 +1,963 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "2ef71144-e2c1-4696-a82a-a9c49dae001e",
+   "metadata": {},
+   "source": [
+    "# Creating Video Clips\n",
+    "This notebook demonstrates using a YOLOv4 model to process individual frames and use the resulting output to generate information about the video.\n",
+    "We use detections of labels over sequential frames to generate Clips which describe the existance of those objects within a specific portion of the video."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8835379a-1c02-44d3-930c-f321279ee0bb",
+   "metadata": {},
+   "source": [
+    "# Install ApertureDB\n",
+    "First we install the ApertureDB python module and other modules we need for running our model. Then, we verify our connection."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d444a543-c419-4d24-8db4-1fea7ccdc270",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install aperturedb tqdm 2>&1 >/dev/null\n",
+    "from aperturedb import Utils\n",
+    "c = Utils.create_connector()\n",
+    "\n",
+    "u = Utils.Utils(c)\n",
+    "u.summary()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7e1a96c2-b0d1-47b6-b836-c0eb0142101f",
+   "metadata": {},
+   "source": [
+    "# Download resources\n",
```

Comment:
COuld you please introduce it as sections / subsections -> right now they are all main headers - so Setup can be main section then Download a subsection etc. This is going to land in docs and so thinking about the look there.